internal/tagging/tagging.go

```diff
@@ -59,6 +59,19 @@ func HasResourceTagFn(resourceType string) bool {
 	return resourceTypeToFnMap[resourceType] != nil
 }
 
+func HasTags(args TagBlockArgs) (bool, error) {
```

Comment:
Setting to beta because this function may need to handle more use-cases.
Basically just checks that the resource has tags.